### DISCUSSION TITLE
Information on how FOSS is being handled from a legal perspective within contracts

### DISCUSSION THREAD

**Discussion by anajsana** [todogroup/ospology#95] [OSPO Discussions]
*Question raised via [TODO Slack](https://thetodogroup.slack.com/archives/C0GBYUWCB/p1650486588230229)*

Does anyone have any information on how FOSS is being handled from a legal perspective within contracts? Is there a group for attorneys to discuss the current state of affairs for FOSS and legal?

**Comment by anajsana** [comment]
*Sean McGinnis replied*

There should be a few folks here that can help, but just wanted to point out if you have a legal team that needs to become more aware of FOSS, LF has this yearly event that may be useful: https://events.linuxfoundation.org/lf-legal-summit/

**Comment by anajsana** [comment]
*Kevin P.Fleming replied*

The FSFE (Free Software Foundation Europe) also has an active and well-populated legal issues mailing list

**Reply by anajsana** [reply]
*Michael Gouwens asked*

Are there mailing lists for the US and/or other parts of the world that you are aware of as well?

**Reply by anajsana** [reply]
*Kevin P.Fleming replied*

Not that I'm aware of; there are some lists where licenses themselves are discussed, but discussion of contracts and their relationship to FOSS wouldn't be on topic on those lists. if Mike Dolan is around he'd know for sure.

**Comment by anajsana** [comment]
*Justin Rackliffe replied*

If you were just talking about how FOSS is called out in contracts my gut says most orgs have clauses for patching and notification, appropriate use, etc. as attestation.  we had some BOM to be provided language that allowed for some verifiability on our side where appropriate.  I am hoping that we will start getting in-toto style attestations, but that's a bit idealistic.

**Comment by anajsana** [comment]
*Chris Aniszczyk replied*

LF has a legal network and legal summit they run usually every year (e.g., https://events.linuxfoundation.org/archive/2020/lf-legal-summit/) - if your company has lawyers, reach out to me and I can get them plugged into the private lists LF has for that

**Comment by itprofjacobs** [comment]
Depending on what they’re looking for, here are some possibilities.  All below are lawyers.

James Vasile of the Open Source Strategies consulting firm is good and FOSS focused.

Heather Meeker does big corporate cases in FOSS but is also big into Open Core, which might be an issue for some folks.

 Karen Sandler of the Software Freedom Conservancy and Even Moglin at Columbia and the Software Freedom Law Center might also be good. Both Free Software focused.

**Comment by debbryant** [comment]
For attorneys I’d also add Andy Updegrove of Gesmer Updegrove LLP; at the same firm and also doing LF work on community codes of conduct and implications for corporations is Joanna Lee.

Pam Chestek does oss work and is considered an expert on trademarks esp. related to oss.

LF list is a good recommendation for just plugging in.

Deb
